src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+class Rectangle extends Figure {
+    private int rightSide;
+    private int topSide;
+
+    public Rectangle(String color, int topSide, int rightSide) {
+        super(color);
+        this.rightSide = rightSide;
+        this.topSide = topSide;
+    }
+
+    @Override
+    public void draw() {
+        StringBuilder builder = new StringBuilder();
+        builder.append("Figure: the top side is of length: ")
+                .append(topSide)
+                .append(" the right side is of length: ")
+                .append(rightSide)
+                .append(" and the color is ")
+                .append(color);
```

Comment:
Consider storing the constructed message in a separate method to separate the concerns of building and displaying the message.